lib/l10n/app_pt_PT.arb

```
@@ -382,6 +382,21 @@
     "taskReminder": "Lembrete de Tarefa",
     "taskReminderBody": "A tarefa \"{taskName}\" está para vencer em {dueDate}",
 
+    "@_": {"description": "Task Budgets"},
```

RuboCop (1.73)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)